devel/universal-ctags/Portfile

```
@@ -3,10 +3,10 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        universal-ctags ctags feffe43a4a9d339cd6701744589746e2297e4567
+github.setup        universal-ctags ctags 0837c42c3fea8854499cf1d22f2f5de12d745f20
 name                universal-ctags
-version             20200111
-revision            1
+version             p5.9.20201213.0
+revision            3
```

When the version increases, the revision drops to 0.
```suggestion
revision            0
```